apps/src/music/views/MiniMusicPlayer.tsx

```diff
@@ -102,6 +104,26 @@ const MiniPlayerView: React.FunctionComponent<MiniPlayerViewProps> = ({
     return <div>Loading...</div>;
   }
 
+  const getPackImageBackgroundStyle = (packId: string) => {
+    const rgb = MusicLibrary.getInstance()?.getPackImageRgb(packId);
+
+    if (!rgb) {
+      return undefined;
+    }
+
+    return `radial-gradient(rgb(${rgb.r}, ${rgb.g}, ${rgb.b}), rgb(0,0,0))`;
```

Comment:
Non-blocking, and this is fine as is, but...

part of me feels like this should be broken apart into at least two methods - the intent feels a little buried. So we've got `getPackImageBackgroundStyle`, but that's always _only_ going to return a radial gradient, and it's also always _only_ going to return a radial gradient that goes to black.

I feel like it'd be better if we had a separate `getRadialGradient` function somewhere:

```
function getRadialGradient(from : RGB, to : RGB) {
  return `radial-gradient(rgb(${from.r}, ${from.g}, ${from.b}), rgb(${to.r}, ${to.g}, ${to.b}))`;
}
```

and then the return in this function is:
```
return getRadialGradient(rgb, {r : 0, g : 0, b : 0})
```

It's a little overengineered in this case, but it's still separating out the functionality - so we have the parent function that gets whatever the background style is from a `packId`, and then a helper function that gets gradients given `RGB` input objects.

Anyway, again, nothing urgent.

Reply:
I like it.  Done.